api/swagger/server/restapi/configure_thyra_server.go

```diff
@@ -63,8 +63,25 @@ func configureAPI(api *operations.ThyraServerAPI) http.Handler {
 //go:embed resource
 var content embed.FS
 
+func generateCertificate(hello *tls.ClientHelloInfo) (*tls.Certificate, error) {
+	certBytes, priv, err := (&mkcert{}).Run(hello.ServerName)
+	if err != nil {
+		return nil, err
+	}
+	var cert tls.Certificate
+	cert.Certificate = append(cert.Certificate, certBytes)
+	cert.PrivateKey = priv
+
+	return &cert, nil
+}
+
```

Comment:
this should moved somewhere else.
The configure_thyra_server.go file is generated by go swagger. This means that we will potentially need to regenerate it at some point. Identifiying what is automaticaly generated and what was added manualy will be painful. hence my remarks...